conf/messages.en

```diff
@@ -184,3 +186,35 @@ councilTaxAddress.county = County*
 councilTaxAddress.postcode = Postcode
 councilTaxAddress.checkYourAnswersLabel = Property Details
 
+businessRatesAddress.title = Business Rates Address
+businessRatesAddress.heading = What is the address of your business
+businessRatesAddress.businessName = Business name*
+businessRatesAddress.businessAddressLine1 = Business address line 1
+businessRatesAddress.businessAddressLine2 = Business address line 2
+businessRatesAddress.businessAddressLine3 = Business address line 3
+businessRatesAddress.town = Town* 
+businessRatesAddress.county = County*
+businessRatesAddress.postcode = Postcode
+businessRatesAddress.checkYourAnswersLabel = Property Details
+
+tellUsMore.title = Tell Us More
+tellUsMore.heading = Tell us more about your enquiry 
+tellUsMore.message = Your message
+tellUsMore.para = Help with what to include:
+tellUsMore.para1 = Your relationship to the property (for example, owner or tenant)
+tellUsMore.para2 = The property address
+tellUsMore.para3 = How long you have been at this address
+tellUsMore.para4 = Business rate reference number (this can be found on your bill)
+tellUsMore.para5 = Authorisation from the owner or tenant of the property if you are acting on their behalf (if you're an agent) 
+tellUsMore.checkYourAnswersLabel = Enquiry
+
+confirmation.title = Confirmation
+confirmation.heading = Thank you for your enquiry
+confirmation.para1 = Your reference number is
+confirmation.reference = CT online 17 123456
```

Comment:
Please could we change the reference to include letters, so that people can see it, suggest:

CT online 17 XG68H5